### DISCUSSION TITLE
Auto close after video ended

### DISCUSSION THREAD

**Discussion by buretvoj** [trizen/youtube-viewer#417] [Q&A] [answered]
Hello, is there a way to auto-close youtube-video from the bash after played audio/video ended? Thanks in advance for the help.

**Comment by trizen** [comment, marked as answer]
One way would be to use `--no-I`.

For example, in selecting the 1st result to play, we would input: `1 --no-I`. After the video played, youtube-viewer will close automatically.

The same option can be used as a command-like argument:

```console
youtube-viewer --no-I [youtube-url]
```

**Reply by buretvoj** [reply]
This is exactly what I am looking for! Thanks a lot for extra-fast response :)
